app/actions/sessions.js

```diff
@@ -159,12 +159,13 @@ export function createScheduledQuery(connectionId, payload = {}) {
       'queries',
       'POST',
       'createScheduledQueryRequest',
-      payload.filename,
+      payload.fid,
```

Comment:
is this correct? My understanding is that scheduled queries only get an `fid` after they are created.

Reply:
my understanding matches @n-riesco's ...

Reply:
:stuck_out_tongue_winking_eye: sometimes my writing is too formal! Hope the message is clear. If I'm reading this code correctly, `createScheduledQuery` will be invoked with  an undefined `fid`. This will set `createScheduledQueryRequest.id` to `undefined`. Is that OK?